core/src/test/scala/unit/kafka/tools/DumpLogSegmentsTest.scala

```diff
@@ -105,7 +105,9 @@ class DumpLogSegmentsTest {
 
   @AfterEach
   def tearDown(): Unit = {
-    log.close()
+    if (log != null) {
+      log.close()
```

Comment:
You can possibly use Utils.closeQuietly here

Reply:
Yes, that makes much more sense. Thank you @divijvaidya!